unstructured/staging/base.py

```diff
@@ -152,6 +154,29 @@ def elements_to_json(
     return json_str
 
 
+def elements_to_ndjson(
```

Comment:
I've always disliked the `elements_to_json()` function because it tries to do two jobs and it makes the calling/typing ugly.

I'd make this two functions, `elements_to_ndjson(elements) -> str` and `elements_to_ndjson_file(elements, filename, encoding) -> None.

And feel free to leave one of those out if you don't have a current need for it.

Reply:
Do we not want to keep with the same pattern? I would think we'd want to split `elements_to_json()` then as well.